### PR TITLE
feat(ui5-combobox, ui5-multi-combo-box, ui5-input, ui5-multi-input): add wrapping in suggestions and n-more

### PR DESCRIPTION
**Pull request by Okiana**
BGSOFUIRILA-3636

### CODE REVIEW

File: packages/main/src/InputPopover.hbs [hunk 1]
```diff
@@ -99,6 +99,7 @@
 				<ui5-li-groupheader data-ui5-key="{{key}}">{{{ this.text }}}</ui5-li-groupheader>
 			{{else}}
 				<ui5-li-suggestion-item
+					wrapping-Type="Normal"
```
Comment (ivoplashkov) [inline packages/main/src/InputPopover.hbs:102]: No need of capital letter for "wrapping-type"

File: packages/main/src/MultiComboBoxPopover.hbs [hunk 2]
```diff
@@ -128,6 +128,7 @@
 
 {{#*inline "listItem"}}
 	<ui5-li
+		wrapping-Type="Normal"
```
Comment (ivoplashkov) [inline packages/main/src/MultiComboBoxPopover.hbs:131]: Same as above

File: packages/main/src/TokenizerPopover.hbs [hunk 3]
```diff
@@ -36,7 +36,10 @@
 		@ui5-item-delete={{itemDelete}}
 	>
 		{{#each _tokens}}
-			<ui5-li .tokenRef={{this}}>
+			<ui5-li
+				.tokenRef={{this}}
+				wrapping-Type="Normal"
```
Comment (ivoplashkov) [inline packages/main/src/TokenizerPopover.hbs:41]: Same as above.

File: packages/main/test/pages/ComboBox.html [hunk 4]
```diff
@@ -262,6 +262,15 @@ <h3>ComboBox in Compact</h3>
 		</ui5-combobox>
 	</div>
 
+	<div class="demo-section">
+		<span>ComboBox - suggestions and show wrapping </span>
+
+		<ui5-combobox class="combobox" id="same-name-suggestions-cb">
+			<ui5-cb-item text="Item that not wrap"></ui5-cb-item>
```
Comment (simonarangelova) [inline packages/main/test/pages/ComboBox.html:269]:
```suggestion
			<ui5-cb-item text="Item that does not wrap"></ui5-cb-item>
```

File: packages/main/test/pages/Input.html [hunk 5]
```diff
@@ -423,6 +423,12 @@ <h3>Input with openPicker() on focusin</h3>
 		<ui5-suggestion-item type="Inactive">No recent users</ui5-suggestion-item>
 	</ui5-input>
 
+	<h3>Input - showing wrapping</h3>
+	<ui5-input show-suggestions id="input-change-wrapping">
+		<ui5-suggestion-item text="Item that not wrap"></ui5-suggestion-item>
```
Comment (simonarangelova) [inline packages/main/test/pages/Input.html:428]:
```suggestion
		<ui5-suggestion-item text="Item that does not wrap"></ui5-suggestion-item>
```

File: packages/main/test/pages/MultiComboBox.html [hunk 6]
```diff
@@ -333,6 +332,13 @@
 			<ui5-mcb-item selected text="Token 1"></ui5-mcb-item>
 			<ui5-mcb-item selected text="Enim do esse anim magna enim fugiat Lorem enim nostrud sit laborum ea."></ui5-mcb-item>
 		</ui5-multi-combobox>
+		<br>
+		<span>MultiComboBox with N-More and show wrapping</span>
+		<br>
+		<ui5-multi-combobox id="mc-more-wrapping">
+			<ui5-mcb-item selected text="Item that not wrap"></ui5-mcb-item>
```
Comment (simonarangelova) [inline packages/main/test/pages/MultiComboBox.html:339]:
```suggestion
			<ui5-mcb-item selected text="Item that does not wrap"></ui5-mcb-item>
```

File: packages/main/test/pages/MultiInput.html [hunk 7]
```diff
@@ -97,6 +97,11 @@ <h1 class="sample-container-title">Multi Input with 11 tokens (overflowing) and
 
 		<br>
 		<br>
+		<h1 class="sample-container-title">Multi Input show wrapping</h1>
+		<ui5-multi-input>
+			<ui5-token slot="tokens" text="Item that not wrap"></ui5-token>
```
Comment (simonarangelova) [inline packages/main/test/pages/MultiInput.html:102]:
```suggestion
			<ui5-token slot="tokens" text="Item that does not wrap"></ui5-token>
```

File: packages/main/src/TokenizerPopover.hbs (hunk 3 above)
Comment (Okiana) [reply packages/main/src/TokenizerPopover.hbs:41]: done

File: packages/main/src/InputPopover.hbs (hunk 1 above)
Comment (Okiana) [reply packages/main/src/InputPopover.hbs:102]: done

**Comment by ivoplashkov** [comment]
Could please merge the master branch, because the test pages keep realoding every second?

File: packages/playground/_stories/main/Input/Input.stories.ts [hunk 8]
```diff
@@ -99,6 +99,16 @@ WithSuggestions.args = {
...
+SuggestionsWrapping.args = {
+	default: `<ui5-suggestion-item text="Wireless DSL/ Repeater and Print Server Lorem ipsum dolar st amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor incidunt ut labore et dolore magna aliquyam erat, diam nonumy eirmod tempor individunt ut labore et dolore magna aliquyam erat, sed justo et ea rebum."></ui5-suggestion-item>
+	<ui5-suggestion-item text="Widescreen Portable DVD Player w MP3, consetetur sadipscing, sed diam nonumy eirmod tempor invidunt ut labore et dolore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergen, no sea takimata. Tortor pretium viverra suspendisse potenti nullam. Congue quisque egestas diam in arcu cursus.Rutrum tellus pellentesque eu tincidunt tortor. Nec tincidunt praesent semper feugiat nibh sed"></ui5-suggestion-item>
+	<ui5-suggestion-item text="Portable DVD Player with 9 inches LCD Monitor"></ui5-suggestion-item>`,
+	placeholder: "Enter product",
+	showSuggestions: true,
+};
+
+
 export const WithValueStateMessage = Template.bind({});
```
Comment (ivoplashkov) [inline packages/playground/_stories/main/Input/Input.stories.ts:112]: I think it was discussed with the team to remove "With" prefix from all samples.

Comment (Okiana) [reply packages/playground/_stories/main/Input/Input.stories.ts:112]: This will be handled in another change for refactoring

**Pull request merged**